### ISSUE TITLE
conflict with Lithium

### ISSUE THREAD

**Issue by iammattcoleman** [malte0811/FerriteCore#61]
I just updated my server to 4.1.2 and noticed that the FerriteCore's changes to PalettedContainer conflict with Lithium's.

```
[13:56:52] [Server thread/WARN]: Method overwrite conflict for method_12334 in lithium.mixins.json:chunk.no_locking.PalettedContainerMixin, previously written by malte0811.ferritecore.mixin.threaddetec.PalettedContainerMixin. Skipping method.
[13:56:52] [Server thread/WARN]: Method overwrite conflict for method_12335 in lithium.mixins.json:chunk.no_locking.PalettedContainerMixin, previously written by malte0811.ferritecore.mixin.threaddetec.PalettedContainerMixin. Skipping method.
```

I had also noticed this when my server crashed several times after updating to FerriteCore 4.1.0. Here's the crash report:
https://gist.github.com/iammattcoleman/ce3c8b6a062c15b20b40b8d9866fc097

Reverting to 4.0.0 or updating to 4.1.1 fixed those crashes.

Currently, I've rolled back to FerriteCore 4.1.1 to ensure that FerriteCore doesn't interfere with Lithium.

**Comment by malte0811** [comment]
First of all: 4.1.0 is very broken, don't use that version.

The warning on 4.1.2 isn't actually a problem, but you can suppress it by either adding `mixin.chunk.no_locking = false` to the lithium config or setting `smallThreadingDetector` to `false` in the FerriteCore config. I'll prepare a proper fix for the next release though.

**Comment by iammattcoleman** [comment]
I appreciate you taking the time to give me the details/workarounds. This mod is one of the key components that makes it possible for my friends and I to play on my not-that-powerful server. Thanks a lot!

**Comment by malte0811** [comment]
Fixed in 4.2.0.
